stdlib/ballerina-jms/pom.xml

```diff
@@ -0,0 +1,115 @@
+<!--
+  ~ Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
@a5anka 2018

Reply:
This pom was originally created in 2017. Should I change?

Reply:
I don't think we have to change the year if the file was created in last year.

Reply:
:+1: